configs/templates/LogTemplate.proto

```diff
@@ -0,0 +1,56 @@
+// COPY OF https://github.com/istio/api/blob/master/mixer/v1/config/descriptor/log_entry_descriptor.proto
```

Comment:
Shoudl we remove this comment? This is 'inspired by' but is definitely not a copy.